package.json

```diff
@@ -19,7 +19,7 @@
   },
   "homepage": "https://github.com/scality/S3#readme",
   "dependencies": {
-    "arsenal": "github:scality/Arsenal#3ed66c5",
+    "arsenal": "github:scality/Arsenal#44b8de5",
```

Comment:
@jonathan-gramain I updated the dependency hash based on the recent one on here: https://github.com/scality/Arsenal/pull/972/commits